libmachine/mcnutils/os_windows.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 func LocalOSVersion() string {
-	command := exec.Command(`systeminfo`)
+	command := exec.Command(`ver`)
```

Comment:
Any particular reason for the backticks instead of `"`?  Seems a bit odd, probably fine though.